node/resolvers/catalog/index.ts

```diff
@@ -105,7 +106,7 @@ export const queries = {
     const resolvedProducts = await Promise.map(products, product =>
       resolveProductFields(ioContext, product, fields)
     )
-
+  
```

Comment:
Actually, I just noticed there is still this trailing whitespace here (there were 4 whitespace chars in the previous review; now there are 2; `master` has none at this line)